doc/nrf/ug_bootloader.rst

```diff
@@ -7,212 +7,185 @@ Secure bootloader chain
    :local:
    :depth: 2
 
-The |NCS| provides a secure bootloader solution based on the chain of trust concept.
+The core architecture behind the secure bootloader solution provided by |NCS| is based on the *chain of trust* concept.
 
-By using this bootloader chain, you can easily ensure that all code that is executed is authorized and your application is protected against running altered code.
-If, for example, an attacker attempts to modify your application, or you have a bug in your code that overwrites parts of the application flash, the secure bootloader chain detects that the flash has been altered and your application does not start up.
+By using this secure bootloader chain, you can easily ensure that all code that is executed is authorized and your application is protected against running altered code.
```

Comment:
```suggestion
By using this secure bootloader chain, you can ensure on the one hand that all code that is executed is authorized and, on the other hand, that your application is protected against running altered code.
```
Had to reread this three times because of the concatenation of "that" and "is", so I'm suggesting a small change.

Reply:
Reworded from your suggestion. Please see latest commit.